lib/transformers/transformers.js

```diff
@@ -127,8 +127,20 @@ const mapInRes = (res, req, startedAt, reqId, opts) => {
   const headers = extractHeaders(opts.headersRegex, opts.headerValueCallback, res.headers);
   const hostFieldName = opts.hostFieldName || 'host';
 
+  const shouldObfuscatePath =
```

Comment:
🔧 either move the obfuscation from here into https://github.com/Workable/riviere/pull/111/files#diff-ba62a6220cc9469f494ea70799192fe379bd233168a9e90ef8a9bbc3de6cda9cR60.

Or move the changes in the previous pr in this file

Reply:
agree